console/src/main.rs

```diff
@@ -60,11 +72,25 @@ async fn main() -> color_eyre::Result<()> {
                 .split(f.size());
 
             let header_block = Block::default().title(vec![
-                Span::raw("connected to: "),
-                Span::styled(
-                    target.as_str(),
-                    Style::default().add_modifier(Modifier::BOLD),
-                ),
+                Span::raw("connection: "),
+                match tasks.connection_state() {
+                    ConnectionState::Connected(t) => Span::styled(
+                        t.as_str(),
+                        Style::default()
+                            .add_modifier(Modifier::BOLD)
+                            .fg(Color::Green),
+                    ),
+                    ConnectionState::Disconnected => Span::styled(
+                        "DISCONNECTED",
+                        Style::default().add_modifier(Modifier::BOLD).fg(Color::Red),
+                    ),
+                    ConnectionState::Pending => Span::styled(
+                        "PENDING",
+                        Style::default()
+                            .add_modifier(Modifier::BOLD)
+                            .fg(Color::Yellow),
+                    ),
+                },
```

Comment:
nit, take it or leave it: it might be worth keeping the remote address in all states, and adding disconnected/pending in addition to the address. for example
```rust
connection: 127.0.0.1:6669 (DISCONNECTED)
```
rather than
```
connection: DISCONNECTED
```

that way, if i have multiple consoles open connected to different instances of a program, i know which is which even if one of them is disconnected.